ACHNBrowserUI/ACHNBrowserUI/views/dashboard/DashboardCollectionProgressSection.swift

```diff
@@ -30,15 +30,32 @@ struct DashboardCollectionProgressSection: View {
         }
     }
     
+    private func makeArtProgressView(icon: String, art: [Item]) -> some View {
```

Comment:
And so because you reuse the `caughIn` function you can remove this whole function and use `makeProgressView` instead :)